manifests/init.pp

```diff
@@ -4,8 +4,11 @@
 # menuing system with syslinux and some pxe tools installed.
 #
 class pxe (
-  $tftp_root='/srv/tftp'
-){
-  include pxe::syslinux
+  $tftp_root        = $pxe::params::tftp_root,
+  $syslinux_archive = undef
+) inherits pxe::params {
+  class { 'pxe::syslinux':
```

Comment:
I'm not super excided about having a class definition here to pass the hardcode.  Could we perhaps instead have the `pxe::syslinux` class require the `pxe` class and just reference the `pxe::syslinux_archive` param or something similar?